sphinx/source/docs/user_guide/geo.rst

```diff
@@ -30,10 +30,11 @@ Bokeh's ``GeoJSONDataSource`` can be used almost seamlessly in place of Bokeh's
     geo_source = GeoJSONDataSource(geojson=geojson)
 
     p = figure()
-    p.circle(alpha=0.9, source=geo_source)
+    p.circle(x='x', y='y', alpha=0.9, source=geo_source)
```

Comment:
As to my knowledge how data specs work, this shouldn't be necessary. But maybe this changed recently (changes by Havoc).